Upgrade postgres app db version (#50310)
metabase/metabase: docs/installation-and-operation/configuring-application-database.md

## PostgreSQL

> We recommend that you use [PostgreSQL](https://www.postgresql.org/) for your Metabase application database. The minimum version we support is 12, but we always suggest to be on the latest version you can.

Reviewer: ```suggestion
We recommend that you use [PostgreSQL](https://www.postgresql.org/) for your Metabase application database. The minimum Postgres version Metabase supports is 12, but we always recommend that you use the latest Postgres version that you can.
```

Does this edit work?
